app/assets/stylesheets/local/buttons.scss

```diff
@@ -25,6 +25,9 @@
   background:transparent;
   color:#2b8cc4;
 }
+.button.button-secondary:focus{
+  color:#005ea5;
```

Comment:
I wonder if there's a variable we can use for this? Maybe try `$link-colour`.

Reply:
We might be able to remove the double class definition too. So just define it as `.button-secondary:focus`. Or even just apply it to all elements with `.button` class?

Reply:
Added in the changes we made when pairing :)